src/apify_client/clients/resource_clients/request_queue.py

```diff
@@ -0,0 +1,152 @@
+from typing import Any, Dict, Optional
+
+from ..._errors import ApifyApiError
+from ..._utils import _catch_not_found_or_throw, _parse_date_fields, _pluck_data
+from ..base.resource_client import ResourceClient
+
+
+class RequestQueueClient(ResourceClient):
+    """Sub-client for manipulating a single request queue."""
+    def __init__(self, *args: Any, client_key: str = None, **kwargs: Any) -> None:
+        """Initializes the RequestQueueClient.
+
+        Args:
+            client_key (str): A unique identifier of the client accessing the request queue
+        """
+        super().__init__(*args, resource_path='request-queues', **kwargs)
+        self.client_key = client_key
```

Comment:
Why the client key is used in `request queue client` but not e.g. in `key value store client`?

Reply:
"Client" here does not mean the API client specifically, but anything accessing the request queue, it's used so that the request queue can differentiate between different applications accessing it in parallel, so each application (client) uses it's own key for that.